tests/main/snap-handle-link/task.yaml

```diff
@@ -12,10 +12,14 @@ prepare: |
 
     # Make sure there is a default handler to open a snap package
     tests.session -u test exec mkdir -p /home/test/.config
+    tests.session -u test exec mv /home/test/.config /home/test/.config.bak
+    tests.session -u test exec mkdir -p /home/test/.config
```

Comment:
This line is not needed anymore.

Reply:
Just making sure there's a .config directory in case something here needs it.

Reply:
Exactly the same thing is done in line 14.